types/jest/index.ts

```diff
@@ -153,6 +153,11 @@ declare namespace jest {
      * for the specified module.
      */
     function setMock<T>(moduleName: string, moduleExports: T): typeof jest;
+     /**
+      * Set the default timeout interval for tests and before/after hooks in milliseconds.
+      * Note: The default timeout interval is 5 seconds if this method is not called.
+      */
+    function setTimeout(timeout: number): typeof jest;
```

Comment:
The documentation comment got indented by one extra space in your last commit. Do you mind fixing that? 😄